packages/mask/src/social-network-adaptor/twitter.com/utils/selector.ts

```diff
@@ -30,7 +30,9 @@ export const searchProfileTabListLastChildSelector: () => LiveSelector<E, true>
 }
 
 export const searchProfileTabPageSelector: () => LiveSelector<E, true> = () =>
-    querySelector<E>('[data-testid="primaryColumn"] [role="navigation"] ~ div [role="heading"] ~ div[aria-label]')
+    querySelector<E>(
+        '[data-testid="primaryColumn"] [role="navigation"] ~ [aria-labelledby^="accessible-list"] [role="heading"] ~ div[aria-label]',
```

Comment:
Actually twitter changed it from `div` to `section`, use attribute selector should be safer here.